ui/app/components/selectable-card.js

```diff
@@ -0,0 +1,29 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+/**
+ * @module MetricsSelectableCard
+ * MetricsSelectableCard components are used to...
+ *
+ * @example
+ * ```js
+ * <MetricsSelectableCard @requiredParam={requiredParam} @optionalParam={optionalParam} @param1={{param1}}/>
+ * ```
+ * @param {object} requiredParam - requiredParam is...
+ * @param {string} [optionalParam] - optionalParam is...
+ * @param {string} [param1=defaultValue] - param1 is...
+ */
+
+export default Component.extend({
+  cardTitleComputed: computed('type', function() {
```

Comment:
is `type` still relevant to the computed property?

Reply:
You're right.  Computed properites observe any changes in the properties they depend on.  Here it makes more sense to have total and not type.  Amended.